src/GraphQL.Tests/TypeExtensionsTests.cs

```diff
@@ -32,6 +33,12 @@ public void is_nullable_using_nullable_types(Type type)
             TypeExtensions.IsNullable(type).ShouldBeTrue();
         }
 
+        [Fact]
+        public void will_throw_on_open_generic()
```

Comment:
And one more - please move this test to src\GraphQL.Tests\Utilities\TypeExtensionsTests.cs

Reply:
You also mind to rename it to GetGraphTypeFromTypeTests.cs to avoid ambiguity.

Reply:
I did move test and rename file, the only thing I did not touch all other tests, this file has half tests named in snake case and half in something between pascal and snake case

Reply:
OK